lib/editor/actions/editor.js

```diff
@@ -31,6 +31,10 @@ import type {
 
 import {clearGtfsContent, saveActiveGtfsEntity, setActiveGtfsEntity} from './active'
 
+const LOCK_API_PATH = '/api/editor/secure/lock'
+/** Identifies what is being locked. */
+const GTFS_EDITOR_LOCK = 'gtfs-editor'
```

Comment:
The item parameter can basically be whatever, right? As long as the clients always use the same value (like gtfs-editor) then it'll work. So perhaps there should be some enum or const array that limits the possible values for this parameter. (I haven't looked at the GTFS+ locking PR yet, sorry if this is done there)